src/Bowl.js

```diff
@@ -0,0 +1,27 @@
+
+function Bowl(frame, roll1, roll2, score, bonus, frameTotal) {
+  this.frame = frame;
+  this.roll1 = roll1;
+  this.roll2 = roll2;
+  this.score = score;
+  this.bonus = []
+  this.frameTotal = 0
+};
+
+
+Bowl.prototype.add = function(roll1, roll2) {
+  this.score += (roll1 + roll2)
+  this.frame += 1
+  this.frameTotal += (roll1 + roll2)
+};
+
+Bowl.prototype.bonusCalc = function(roll1, roll2, frame, score) {
+  if (this.frame == 10 && this.score == 0) {
+    this.bonus.push("Gutter")
+  } else if (this.frameTotal == 10) {
+    this.bonus.push("Strike")
+    this.frameTotal = 0
+  } else if (this.bonus[-1] == "Strike") {
```

Comment:
javascript doesn't know '-1', instead, use this.bonus[this.bonus.length - 1] or this.bonus.last

pls refer to http://www.codeblocq.com/2016/05/Get-the-last-element-of-an-Array-in-JavaScript/

Reply:
in Javascript (because it's erm a quite peculiar language) it's always better to use a triple equal sign (===) rather than (==) 

https://codeburst.io/javascript-double-equals-vs-triple-equals-61d4ce5a121a
https://bytearcher.com/articles/equality-comparison-operator-javascript/